crates/tauri-cli/src/mobile/ios/mod.rs

```diff
@@ -232,7 +227,11 @@ pub fn get_config(
             }
           }
         }),
-    ios_features: ios_options.features.clone(),
+    ios_features: if ios_options.features.is_empty() {
+        None
+    } else {
+        Some(ios_options.features.clone())
+    },
```

Comment:
```suggestion
    ios_features: Some(ios_options.features.clone()),
```

leftover, https://github.com/tauri-apps/tauri/pull/14607/files#r2649702462